### PR TITLE
cpu/stellaris: fix header include guards.

### PR DESCRIPTION
**Pull request by zhuoshuguo**
Pull request created as per the issue  [#2623](https://github.com/RIOT-OS/RIOT/issues/2623#)

### CODE REVIEW

**Comment by gebart** [comment]
I think some of the stellaris_common include guards will clash with the RIOT periph include guards. Could you do a common prefix on all stellaris_periph files like you did with the cpu.h?
`STELLARIS_BLAH_H_`

The stellaris periph headers are included with a path prefix, like `#include "stellaris_periph/uart.h"`, because of the way the cflags are set up, so it makes sense to add a prefix to the include guard as well.

Also, it's better to change them all now than to change some of them and discover later that we have clashes when we implement more features.

File: cpu/stellaris_common/include/stellaris_periph/uart.h [hunk 1]
```diff
@@ -37,8 +37,8 @@
 //
 //*****************************************************************************
 
-#ifndef __UART_H__
-#define __UART_H__
+#ifndef UART_H_
```
Comment (aabadie) [inline cpu/stellaris_common/include/stellaris_periph/uart.h]: As reported by Murdock, you have a trailing white space here.

**Comment by aabadie** [comment]
General comment: in the initial issue @haukepetersen pointed out that the header guards fix should be made in small PRs #2623. It's obviously not the case here.

I would advise to split this PR is smaller ones, e.g one for each cpu subfolders affected (stellaris_common, stm32f* and x86). It makes reviewing easier and avoids potential conflicts with other PRs.

**Comment by zhuoshuguo** [comment]
Adopted comments, updated. Please check.

**Comment by gebart** [comment]
ACK & go

**Pull request merged**